docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -2179,6 +2190,43 @@ Model Limitations
   fortified food is the same as their mother's. It may be better to model these
   propensities as positively correlated but not necessarily identical.
 
+* For the iron effect on birthweight:
+
+  - We are assuming that flour consumption is independent of birthweight as
+    sampled by GBD. Thus we are not accounting for confounding by overall
+    caloric intake, which is likely correlated with both flour consumption as
+    well as birthweight.
+
+  - We are also assuming that fortification is independent of flour
+    consumption, given whether or not you eat flour. Thus we are not accounting
+    for confounding due to factors like urban vs. rural settings, which could be
+    correlated with both flour consumption and fortification.
+
+  - Ideally, we would want to get correlations between birthweight, flour
+    consumption, and fortification in our populations, so that we could create a
+    joint distribution to appropriately sample all three variables at once.
+    Accounting for this correlation would likely reduce the impact from
+    increased birthweight.
+
+* GBD data for neural tube defects is too low, so we are likely underestimating
+  the impact of folic acid fortification.
```

Comment:
I think we should add more detail/information here. 
```suggestion
* GBD data for neural tube defects, particularly for India, is low relative to estimates of NTD burden from other data sources. Therefore, we are likely underestimating
  the impact of folic acid fortification in our simulation that relies upon GBD data.
```

Reply:
Thanks for the suggestion. I made this change and added citations.